### PR TITLE
Implement boto3 flow for non amd archs

### PR DESCRIPTION
**Pull request by mkumatag**
- use argparse
- use boto3 sdk for non-amd64 arch

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/13#" title="Author self-approved">mkumatag</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: scripts/images/upload_image.py
Comment (bpradipt) [inline scripts/images/upload_image.py:50]: Possible to add a system arch check to ensure it only gets used for Intel ?

Comment (mkumatag) [reply scripts/images/upload_image.py:52]: fixed, ptal

**Review by mkumatag — COMMENTED** [review]
fixed

**Review by bpradipt — APPROVED** [review]
/lgtm
Thanks @mkumatag

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/infra/pull/13#pullrequestreview-493122428" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/infra/pull/13#" title="Author self-approved">mkumatag</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](?repo=ocp-power-automation%2Finfra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
